third_party/npm_@brave_swap-interface/README.chromium

```diff
@@ -0,0 +1,3 @@
+Name: @brave/swap-interface
+URL: https://github.com/brave/swap
```

Comment:
seems this repo needs some security stuff set up, ex: https://github.com/brave/swap/pull/100

i also recommend a github action that runs `npm audit`

cc @kdenhartog 

probably both the swap repo and this PR should have a security review (can be the same ticket)

Reply:
Yeah a security ticket to manage this would be good. Please also assign it to me @onyb

Reply:
Done: https://github.com/brave/security/issues/1100 

@kdenhartog